File: MPI/mpicheck/tests/functions/2.0/MPI_Status_set_elements.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Status *var_0;
    MPI_Datatype var_1;
    int var_2;
    int ret;
    /* calls */
    ret = MPI_Status_set_elements(var_0, var_1, var_2);
    ret = PMPI_Status_set_elements(var_0, var_1, var_2);
    return 0;
}

File: MPI/mpicheck/tests/functions/2.0/MPI_Status_set_elements.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0(10)
       INTEGER var_1
       INTEGER var_2
       INTEGER var_3
        call mpi_status_set_elements(var_0, var_1, var_2, var_3)
       call pmpi_status_set_elements(var_0, var_1, var_2, var_3)
        end program main
    